packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -113,11 +113,11 @@ const useStyles = <StyleProps extends PrimitiveProps>(
     allDisplayNames: composeOptions?.displayNames || [displayName],
     className: composeOptions?.className || className,
     primaryDisplayName: composeOptions?.displayName || displayName,
-    props: {
+    componentProps: {
```

Comment:
Would recommend mentioning in the PR description, that the changes are referring to the `getStyles` function.

Reply:
Done 👍